index.js

```diff
@@ -1,21 +1,86 @@
-document.getElementById('open-bar').onclick = function() {
-    document.getElementById('menu-bar').classList.add('show')
-    document.getElementById('open-bar').classList.add('close');
-    document.getElementById('close-bar').classList.remove('close');
-    // document.getElementById('navi').classList.add('navi');
-    
-}
-
-document.getElementById('close-bar').onclick = function(){
-    document.getElementById('menu-bar').classList.remove('show')
-    document.getElementById('open-bar').classList.remove('close');
-    document.getElementById('close-bar').classList.add('close');
-    // document.getElementById('navi').classList.remove('navi');
-}
-
-document.getElementById('toggle').onclick=function(){
+/* eslint-disable prefer-destructuring */
+document.getElementById('open-bar').onclick = () => {
+  document.getElementById('menu-bar').classList.add('show');
+  document.getElementById('open-bar').classList.add('close');
+  document.getElementById('close-bar').classList.remove('close');
+};
+
+document.getElementById('close-bar').onclick = () => {
```

Comment:
Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/210367096-a3cd58ec-9b2c-48ee-9a56-f5ca205002f6.png"> | <img src="https://user-images.githubusercontent.com/69011963/210367134-877f6cf6-a65d-46ee-b3ac-e7d0987ee306.png" width="1440"/> 

- [ ] 1. Your stack is not in line with the original design parameters; please adjust its dimensions accordingly. 🔧 
- [ ] 2. Please reduce the size of the close button. ❎
- [ ] 3. Please ensure that the font size matches that of the original design. 📐
- [ ] 4. The "See Live" and "See Sources" buttons are not positioned by the initial design, kindly rectify this where necessary. 📝
- [ ] 5. Kindly eliminate any superfluous, wide empty spaces at the bottom of the page. 🪐